code/WorkInProgress/recycling/disposal_chute.dm

```diff
@@ -136,18 +136,31 @@
 
 	// mouse drop another mob or self
 	//
-	MouseDrop_T(mob/target, mob/user)
+	MouseDrop_T(atom/target, mob/user)
 		//jesus fucking christ
-		if (!istype(target) || target.buckled || get_dist(user, src) > 1 || get_dist(user, target) > 1 || is_incapacitated(user) || isAI(user) || isAI(target) || isghostcritter(user))
+		if (!in_interact_range(src,user) || !in_interact_range(src,target) || isAI(user) || is_incapacitated(user) || isghostcritter(user))
```

Comment:
kind of silly but pretty sure this would let people with old tk remotely dump someone else also with remote tk into the trash

Reply:
okok, changed the checks to use IN_RANGE instead